crates/storage/nippy-jar/src/filter/mod.rs

```diff
@@ -0,0 +1,39 @@
+use crate::NippyJarError;
+use serde::{Deserialize, Serialize};
+
+mod cuckoo;
+pub use cuckoo::Cuckoo;
+
+/// Membership filter set trait.
+pub trait Filter {
```

Comment:
naming nit: can we rename this to something that conveys what this is used for `<>Filter`?

Reply:
`InclusionFilter` ?  this is hard